### PR TITLE
fix!: validate initial height and set default values for init consumer params 

### PR DESCRIPTION
**Pull request by mpoke**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [ ] Included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] Added `!` to the type prefix if the change is [state-machine breaking](https://github.com/cosmos/interchain-security/blob/main/RELEASES.md#breaking-changes)
* [ ] Confirmed this PR does not introduce changes requiring state migrations, OR migration code has been added to consumer and/or provider modules
* [ ] Targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
* [ ] Provided a link to the relevant issue or specification
* [ ] Followed the guidelines for [building SDK modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/00-intro.md)
* [ ] Included the necessary unit and integration [tests](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#testing)
* [ ] Added a changelog entry to `CHANGELOG.md`
* [ ] Included comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] Updated the relevant documentation or specification
* [ ] Reviewed "Files changed" and left comments if necessary 
* [ ] Confirmed all CI checks have passed
* [ ] If this PR is library API breaking, bump the go.mod version string of the repo, and follow through on a new major release

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` the type prefix if the change is state-machine breaking
* [ ] confirmed this PR does not introduce changes requiring state migrations, OR confirmed migration code has been added to consumer and/or provider modules
* [ ] confirmed all author checklist items have been addressed
* [ ] reviewed state machine logic
* [ ] reviewed API design and naming
* [ ] reviewed documentation is accurate
* [ ] reviewed tests and test coverage

### CODE REVIEW

File: x/ccv/provider/types/msg_test.go [hunk 1]
```diff
@@ -647,3 +647,57 @@ func TestMsgAssignConsumerKeyValidateBasic(t *testing.T) {
...
+
+func TestValidateInitialHeight(t *testing.T) {
+	testCases := []struct {
+		name          string
+		chainId       string
+		initialHeight clienttypes.Height
+		expPass       bool
+	}{
+		{
+			name:    "valid with revision number",
```
Comment (MSalopek) [inline x/ccv/provider/types/msg_test.go:659]: Please consider adding a testcase and checking if the code works for evmos-like chain names:
* [evmos_9001-2](https://github.com/cosmos/chain-registry/blob/master/evmos/chain.json#L9)

**Review by MSalopek — APPROVED** [review]
Tentative approval but please check evmos-like chain names.

File: x/ccv/types/params.go [hunk 2]
```diff
@@ -34,10 +34,10 @@ const (
 	// (and for consistency with other protobuf schemas defined for ccv).
 	DefaultHistoricalEntries = int64(stakingtypes.DefaultHistoricalEntries)
 
-	// In general, the default unbonding period on the consumer is one week less
+	// In general, the default unbonding period on the consumer is one day less
 	// than the default unbonding period on the provider, where the provider uses
 	// the staking module default.
-	DefaultConsumerUnbondingPeriod = stakingtypes.DefaultUnbondingTime - 7*24*time.Hour
+	DefaultConsumerUnbondingPeriod = stakingtypes.DefaultUnbondingTime - 24*time.Hour
```
Comment (MSalopek) [inline x/ccv/types/params.go:40]: Why was this changed?

**Comment by MSalopek** [comment]
I'm trying to figure out the revision height problem locally using some externally supplied data. Let's pause this for now (unless you're fixing the tests).

File: x/ccv/types/params.go (hunk 2 above)
Comment (insumity) [reply x/ccv/types/params.go:40]: No unbonding pausing anymore, so I guess no reason for it to be much smaller (?)

**Review by insumity — APPROVED** [review]

File: x/ccv/types/params.go (hunk 2 above)
Comment (mpoke) [reply x/ccv/types/params.go:40]: Indeed
